README.md

```diff
@@ -293,6 +293,16 @@ Plugin 'dense-analysis/ale'
 git clone https://github.com/dense-analysis/ale ~/.vim/bundle/ale
 ```
 
+#### [lazy.nvim](https://github.com/folke/lazy.nvim)
+```lua
+{
+    'desnse-analysis/ale',
+    config = function()
+        -- Configuration goes here.
```

Comment:
Better add some more details, for example how to set  specific linters:

```lua
config = function()
  local g = vim.g

  g.ale_ruby_rubocop_auto_correct_all = 1

  g.ale_linters = {
      ruby = {'rubocop', 'ruby'},
  }
end
```

Reply:
I have updated the config function with an example as a starting point to build upon